src/main/java/org/broadinstitute/hellbender/utils/bwa/BwaMemIndexImageCreator.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.hellbender.utils.bwa;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.BwaMemUtilitiesProgramGroup;
+
+
+/**
+ * Simply creates the reference index image file.
+ *
+ */
+@CommandLineProgramProperties(
+        summary = "Creates the image file for use by BwaMemAligner",
+        oneLineSummary = "Creates the image file for use by BwaMemAligner",
+        programGroup = BwaMemUtilitiesProgramGroup.class
+)
+public final class BwaMemIndexImageCreator extends CommandLineProgram {
+
+    @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
+              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+              doc = "Output file (if not provided, defaults to STDOUT)",
+              optional = false)
+    private String referenceFastaLoc = null;
+
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+              shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+              doc = "Output file (if not provided, defaults to STDOUT)",
+              optional = false)
+    private String referenceIndexImageOutputLoc = null;
+
+    @Override
+    protected final Object doWork() {
+        BwaMemIndex.createIndexImage(referenceFastaLoc, referenceIndexImageOutputLoc);
```

Comment:
@SHuang-Broad @tedsharpe As far as I can see it looks like `createIndexImage` requires that they already ran the `bwa index` command.  

Would it be possible for us to expose that `index` to the jni so that you wouldn't need a separate copy of bwa-mem to create the index.  (It could cause a lot of confusion if there are version differences between the packaged bwa mem and the one that people are indexing with.)